### ISSUE TITLE
favicon requested using incorrect HTTP/S schema

### ISSUE THREAD

**Issue by MrDetonia** [grokability/snipe-it#11945]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When using HTTPS, a custom favicon is requested over HTTP, resulting in the content being blocked by browsers due to mixed HTTPS/HTTP content.

### Reproduction steps

1. Enable HTTPS
2. Upload custom favicon
3. Disable / clear browser cache and log in to Snipe IT
4. Monitor network requests in browser dev tools and see the blocked request.

### Expected behavior

Favicon is requested in a schema that matches the one currently being used, e.g. HTTP when using HTTP, HTTPS when using HTTPS.

### Screenshots

![image](https://user-images.githubusercontent.com/9915250/194838948-eaece86f-6bdf-4f11-babf-741a9e4b5456.png)
![snipeit](https://user-images.githubusercontent.com/9915250/194839428-8950bc91-1e44-4cba-8b4b-3dab6af3cad4.jpg)

### Snipe-IT Version

6.0.10

### Operating System

Debian 11

### Web Server

Apache

### PHP Version

7.4.32

### Operating System

Windows

### Browser

Edge, Chrome, Brave

### Version

Latest

### Additional context

Fresh install from Github. No Docker or container manager etc.

**Comment by snipe** [comment]
We do not ever hard code protocols into your urls, since we have no way of knowing whether you’re running TLS or not.

https://github.com/snipe/snipe-it/blob/fbfc9d123cb1e0dae5c5b22b4a3f2a0b0d0639cb/resources/views/layouts/default.blade.php#L18-L20

This is all dictated by your APP_URL in your .env

**Comment by MoralCode** [comment]
i had a similar issue where some styles were HTTP and others were HTTPS because my snipe install is behind a proxy and something got restarted causing the IP address i had set in `APP_TRUSTED_PROXIES` ([link to docs](https://snipe-it.readme.io/docs/docker#behind-a-ssl--hsts-reverse-proxy)) to be wrong.

**Comment by WimSuenens** [comment]
There's an extra space at the end in the link for the favicon.
```html
<link rel="shortcut icon" type="image/ico" href="{{ ($snipeSettings) && ($snipeSettings->favicon!='') ?  Storage::disk('public')->url(e($snipeSettings->favicon)) : config('app.url').'/favicon.ico' }} "> 
```
Could that be the issue?

**Comment by snipe** [comment]
Hi there - sorry for the freakishly long time in responding here. This one fell off our radar :( 

Since the time this issue was opened, a ton of changes have been made in Snipe-IT. If you're still seeing this issue in the current version, please open a new issue that includes:

- Version of Snipe-IT you're running
- Specific errors you're seeing in the UI and/or in the browser console and error logs
- Steps to reproduce 
- Whether or not you can reproduce this on the demo (https://demo.snipeitapp.com)

Thanks - and again, sorry for the very late reply. We're doing some cleanup of old issues, so we're rediscovering some older things that didn't get addressed the first time around. 

![BeakerEmbarrased](https://github.com/user-attachments/assets/7be8d162-c2a4-46df-b3c6-e3adafc49a51)

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>
